teddy.js

```diff
@@ -387,6 +372,42 @@
 
       renderedTemplate = teddy.templates[template] || renderedTemplate
 
+      // check if noparse or noteddy tag exist
+      dontParse = renderedTemplate.match(/<include[^>]*( noparse| noteddy)[^>]*>/g)
+
+      if (dontParse) {
+        // cache length
+        dontParseLength = dontParse.length
+
+        // if multiple noparse or noteddy exist, push src names into an array
+        for (i = 0; i < dontParseLength; i++) {
+          src = getAttribute(dontParse[i], 'src')
+          srcArray.push(src)
+        }
+        if (!src) {
+          if (teddy.params.verbosity) {
+            teddy.console.warn('<include> element found with no src attribute. Ignoring element.')
+          }
+          return ''
+        } else {
+          for (j = 0; j < srcArray.length; j++) {
```

Comment:
Since this loop stands on its own and isn't nested inside another for loop `j` is not needed and `i` can be used.

Also since you're caching the length in the previous loop that pattern should be followed here as well.